src/main/java/bio/terra/cli/businessobject/Resource.java

```diff
@@ -176,9 +176,9 @@ public void delete() {
   public abstract String resolve();
 
   /**
-   * Check whether a user's pet SA can access a resource.
+   * Check whether a user can access a resource.
```

Comment:
Aren't we still checking that the pet SA can access a resource? I thought we just moved the pet SA credential fetch from the CLI to WSM.

Reply:
That's true. I'm wondering if at this point pets are really an implementation detail in WSM and clients shouldn't know about them.

Reply:
Clients (or at least the CLI) still need to know about pet SAs because they may talk directly to the cloud in some cases (e.g. to get the number of tables in a BQ dataset). It can't use the user's credentials for that, because of insufficient OAuth scopes, so it has to call SAM to get the pet SA credentials.

Relatedly, users also have to know about proxy groups, because they need to share external resources with their proxy group if they want to add a referenced resource.

Reply:
That's good to know. I'll restore the previous value of the comment.